### ISSUE TITLE
Missing word of first colloseum hint NPC / Other various text errors

### ISSUE THREAD

**Issue by Clydefrosch22** [Ich73/DQM2-FanTranslation#12]
In the sentence "If fire off a lucky Curse of RUmba, you'll have a better chance at defeating it!", there's a "you" missing.

if this the right place to report these? one issue per error or should there be one Issue for everyone to report such mistakes?

**Comment by Clydefrosch22** [comment]
Her third line "If launch into a lucky Fuddle Dance, you'll have" is also missing a you

**Comment by Ich73** [comment]
> if this the right place to report these? one issue per error or should there be one Issue for everyone to report such mistakes?

Yes, this is the right place.
If you have a list of multiple problems, you can use one issue for it. Especially if it's with the same NPC.
Once I fixed those problems, I will close the issue. You should then start a new one.

**Comment by Clydefrosch22** [comment]
Alright.
There's a line break error (and a general mistake) when the Monster Farm Service gets introduced
"You can use this option to pull monsters from yours farm, building
an ally and opposing team, to pit them against each other..."

there's an extra s in your farm and the word building leaves the textbox and gets cut off at the n.
Breaks should be after "farm," and then the next line would need a break after "each" for it to work

**Comment by Clydefrosch22** [comment]
Not sure if this is really an error, but what's up with Water and Pirate Key?
It's refered to as pirate key in conversations, but in the item/keydoor menu it's the Water Key?

**Comment by Ich73** [comment]
> Not sure if this is really an error, but what's up with Water and Pirate Key?

We changed some names during development. So, things to fix:
  * `Pirate Key` to `Water Key`
  * `Oasis Key` to `Desert Key`
  * `Ice Key` to `Snow Key`
  * `Oasis World` to `Desert World`

**Comment by Clydefrosch22** [comment]
The "Welcome to Polona village." NPC doesn't capitalize Village.

**Comment by Clydefrosch22** [comment]
Unfortunately, I clicked it away too fast so I don't remember the text, but when the Mother gives you a Weapon as a reward for bringing her 5 Punchroom items, the Weapon name had a weird japanese symbol at the end? I feel like that weird symbol was supposed to be a star?

**Comment by Clydefrosch22** [comment]
In Port Ritz, some of Kamehas lines extends past the textbox
"Rottney already told me all about how you're ALSO seaching for
a replacement for the Navel.
the break should be after "searching"

Ha-ha-ha-ha! I had no clue how to react back then when it initially
happened, so I skipped town..."
Here, the break needs to be after "when it"

"As you can see, I've already gotten myself two henchmen, you
see!?"
Here, the line is just barely in the box, it would probably look better with the break after "henchmen,"

**Comment by Clydefrosch22** [comment]
Mermaid Queen has another one
"I have not spoken with humans ever since those pirates invaded
our home, and stole our treasure."

Break should be after "pirates" and there shouldn't be a comma after home

in the next sentence, "Is it true.... that you have come here in search of
Har's Mirror, as well?" the comma probably also should be removed.

And once more in
"Speak with the mermaid who brought you here to return to the surface, and go visit that altar." 
The comma also shouldn't be there.
If these are meant as hints that she does a lot of pauses, more ... is probably the right way to go about it.

"It seems... there is no left but you who can save our treasure. May your travels be safe." I think it's meant to be no one left but you

**Comment by Clydefrosch22** [comment]
And another from the Mermaid Queen
"In order to those who keep the Moonstone for
themselves, you must meet the Beast of the Sea."
It should probably be In order to find those who.

And the line
"Please speak with the mermaid who brought you here as
she will guide you to the altar."
just reads weirdly. Instead of "as", maybe it should be "and"

**Comment by Clydefrosch22** [comment]
I noticed the Zombies on the pirate ship use a skill called
Dead Obsession
but once it runs out, it says the effect of Tenacity of the Dead ended.

Is it possible that either of those should be renamed?

**Comment by Clydefrosch22** [comment]
Papa on the farm when you bring him the Scarfish has another line break thing

"Make sure to bring it along whenever you go up against a foe that
relies heavily on attacks with curse effects!"
break should be after "a"

**Comment by Clydefrosch22** [comment]
there's a lot of messed up line breaks in the wifi lobby, mostly when the 'hometown' is displayed. Various places have linebreaks, I guess everything with more than 6 Letters.

On ranking lists and when initiating a match, even the word hometown itself seems too long. 
maybe shorten it to just Home

**Comment by Clydefrosch22** [comment]
"Heh-heh-heh. Looks like I win, this time! I got the treasure
first!"
needs to lose the comma too

**Comment by Clydefrosch22** [comment]
I guess you should just check all the 
"If launch into a lucky..."
 sentences, most of them seem to be missing the "you"

**Comment by Clydefrosch22** [comment]
From the Spring Spirit
"You must now east from here, to Norden, and restore peace
to their Kingdom."

missing a "heading", which would move the line break to after "restore".
also not sure, but I feel like kingdom in this case doesn't need to be capitalized

**Comment by Clydefrosch22** [comment]
Linebreak in
"Watch it, comin' through! Make way for the great Prince Kameha
from the Kingdom of Great Log!"

Break should be after "Prince" or "great"

**Comment by Clydefrosch22** [comment]
"The Princess is behaving well right now, despite her depression."

Break should be after "her"

**Comment by Clydefrosch22** [comment]
"Howevery she's been sneaking off and keeping to her self way
more than usual. I wonder what..."

it should be "herself"

"Thanks to you, kid, we just averted making a huge mistake
we our kingdoms never could've made up for."
I guess either remove "we" or "our kingdoms"?

"Firstly, I doubt are the Speaker of the Spirit. Who are
you actually working for?...Are you a Westanian spy!?"

missing a "you" and I believe a space between ? and ...

"Ha-ha-ha-ha! I bet you're shocked to the same guy down
here, that just sent you down here, eh?"

missing a "see", adding it would likely make it necessary to move the line break after "guy". Down here and down here sounds a bit weird repeating itself, but that might've been intended.

"Then, Prince Kameha here came along, and we tried to come
up with an escape plan..."
the second comma should be removed.



not 100% sure about this one, it might just be that one of the Norden soldiers was meant to speak like stereotypical russian? 

"I'm not surprised they have Princess... but what does
that mean, about her smile?"

**Comment by Clydefrosch22** [comment]
In general, there's a couple of instances of ", and", most of which I believe should have the comma removed.

**Comment by Clydefrosch22** [comment]
"I said, Enough! Do not make me repeat myself again..."
enough shouldn't be capitalized

"...9th day of the Rose. Ohhh, I screw up. To think, a 
soldier would be guarding my escape vine."
I screwed up. And I don't think the comma after think needs to be there.

"Princess Sonya's always been quite the tom boy."
tomboy.
I feel like I glanced tom boy a second time at some point, but I didn't remember to make a screenshot at the time.

"I had that scholar brew me up a potion though, and that thing
 knocked me within seconds....'
another ,and, also not sure if "knocked me within seconds" is actually an expression, or if there's a word missing.


"Make pals a monster that has gotten "+25" of the "+
value", and bring it to me.
Make pals with a monster.
the line break, if possible, should be before "+ value" just to it on the same line.
,and


"Make pals with a monster that has 2 skills that increase
defense, and bring it to me!"
2 should be written out as two
,and


"You can get Grim Grinner by breeding a Boss Troll with a Skeleton
Soldier!"
Text extends past frame, needs a line break before "Skeleton"

"Really! Thank you! Then I should be able to climb into
that item bad you're carrying!"
should be item bag

"WHAT!? She looks JUST like me! Wait does that mean I'm..."
needs a comma after Wait.

"Please! You must release me from this bag, at once!"
I think you can drop that comma.

"Fear not, old friend. We too almost turned against our
allies in the west, due to some evil meddlers!"
We, too, almost...
but the comma after west can be removed i think.

"You should return to the Spring Spirit and inform her that your
role as Speaker has been fulfilled."
your just about touches the border, maybe move the break before "your role"


"Thank you. You've rid us of the evil, that had cast its
dark shadow upon out world."
Also think the comma here can be dropped.

There was another mistake in the Spring Spirits lines just before you get the Celestial Shield, some missing or misspelled word, but I skipped it trying to make the screenshot and couldn't remember it unfortunately.


"When that time comes, please do stop back and pay me another
visit."
Line break needs to be moved before "another"

**Comment by Clydefrosch22** [comment]
"You can use the Key at the Mysterious Door in Great Log Castle"
extends beyond box.
Linebreak after Log.

**Comment by Clydefrosch22** [comment]
Another issue mentioned the error when Papa gives you the Thaumatosaurus quest.
When you finish it, there's another linebreak issue, like it tries to display three rows in one box.
Couldn't make a screenshot so I don't remember exact line, but there can't be to many lines with Thaumatosarus in them.

**Comment by Clydefrosch22** [comment]
Every week when the ranked battles are through, the top screen shows a picture saying "Took 1st Place!!"
no matter what place you actually ranked as.
I suppose that is an image and can't be changed dynamically to display the actual rank, so maybe translate it to something more vague or remove the text at all.

**Comment by Clydefrosch22** [comment]
"I have brought that who you have sent for...
Scout Tara, your Majesty!"
maybe drop the 'that'?

"It makes warms my heart that you would stand before
me as a paragon of justice. Your can-do attitude shines through."
drop "makes" and move "me" to the first line because the second line extends the box.

"Now I did not call upon you for nothing. It's about my fool
of a son, Prince Kameha."
comma after now?

"From what I've been told, He...... has followed after you
into the other worlds, correct?
uncapitalize "he" and add a space after it.
also, remembering a lot of ......... in the script so far, have you considered not doing more than 3 dots? Or at least putting a space between ... ... ... them? I think that makes it look nicer.

"He IS the sole heir to this kingdom. He shouldn't be sneaking
off to such dangerous among the other worlds."
such dangers? such dangerous other worlds?

FINE! I'm gonna mine the mountain through and through,
an' This time we'll strike the motherload!"
uncapitalize This.

"Speak with the village chief, who lives on the southside
of this village before setting off."
I think there should be a comma after village.
also, in this instance, I think south side should be separated?

"And then BAM! Just like that, a NEW Demonlord shows up outta
what seems like thin air!"
Linebreak after "up"


"Oh! You've brought a Grim Grinner! Okay! You've cleared Papa's
eighth lesson!"
Linebreak after "cleared"
maybe You've brought me a Grimm Grinner?

"Be patient until you get stronger. This is another importan
tlesson passed down in our family"
that thing with the t

"I mean if ya were to be the next Demonlord..."
comma after mean?

"As the one who has brought which His Darkness has long
sought, I have a message for you from the Demonlord, himself."
I think you can drop the comma after Demonlord.
The f and period also reach beyond the textbox. Maybe drop himself completely. Or take out 'for you', in either case, the sentence still works.

"I am, Murdaw, the Demonlord. You, who have 
surpassed all other monsters......"
I'm not sure about this one.
The first comma seems wrong, but maybe it's supposed to be a reference to Murdaws text in DQ6?
Maybe I am the Demonlord, Murdaw! instead?
And unless it's referential, it should be "You, who has"

"Now, as request, show me the treasures you've collected."
should be "requested"

"It matters not. You've most certainly earned this
Light Ord. Do with it, what you wish."
There was at least one more instance where the Light Orb was refered to as Light Ord.

"Head up, whenever you're ready.  And do remember to
mind your manners in front of our Noble King."
It might just look like it, but there seem to be two spaces before "And"
And I'm not sure if Noble should be capitalized. King, in this context should be capitalized, but I don't think noble?

"Y-Y-You're... It's you, Tara! So you
were the caused of all the ruckus below!?"
"the cause" or "the one who caused"

"Sweet Ol' Murdaw suggested the idea, so that if I ended
up havin' to hand it over, to use a fake."
ol' probably shouldn't be capitalized.
And the sentence seems weird overall.
Maybe 
"Sweet Ol' Murdaw suggested the idea, that if I ended
up havin' to hand it over, I should use a fake."

"I haven't been in quite a sticky situation as this, in
quite some time..."
quite as sticky a situation?
and I think you can drop the comma

"The guard's been awfully strict these days though. But I'll
break free and chase after you."
Is this referring to one singular guard? Or all guards?
If one, "The guard has been awfully strict these days. But..."
If all "The guards have been awfully..."


"It seems that spirit vanished with the life force of the Mistle
Tree was lost as well."
Maybe "It seems that the/this spirit vanished with the life force
of the Mistle Tree and was lost as well."?
I'm not sure about this one.


"Come home as soon as you can because a nice, hot
home-cooked meal waits for you when you get back."
comma after because and hot
potentially also one between you and when?

"I know 'cause I'm a Plant monster. And what I knowm is that
Great Log's life force is all used up.
not sure if Plant needs to be capitalized, are all monster families capitalized?
there's an extra m in know.

"I heard you're settin' off to explore some of the 
Other worlds, eh?"
uncapitalize Other

"I dunno the reason, but at this rate the island's gonna sink!"
comma after rate?

"Mama went at Papa telling him that he can't just run away
from his problems"
comma after Papa.
Is 'went at someone' an expression?

"But it makes me all gooey inside when she's pleased with me
so I'm gonna do my best!"
comma after second me

"I planned on retiring here, but my burning passion for being
a scout been renewed, after watching your recent matches."
"has been renewed"
This text box is already full. adding "has" will require a third line.
and I think you can drop the second comma.


"Well, kiddo! Come at me, and quick! We gotta get this over with,
'cause my ol' bellys already rumblin' from hunger!"
Line break after "over"
not 100% sure, but belly's?

"If nail it with a Furious Slash, you'll have a better chance
at gainined the upper hand against it."
"If you nail it with" (might have to move line break before "chance")
and "at gaining the upper..."

**Comment by Clydefrosch22** [comment]
Right after the main game, when that soldier comes to the farm and you go talk to your mother, she suddenly starts talking in heavy accent. I didn't realize at the time, only once I went back to her in the house, that she usually doesn't do that? I mean, occasionally she says like yer instead of your or drops a g at the end of a word, but she really turned it up to eleven in that scene.

**Comment by Clydefrosch22** [comment]
When you get Darck's Orb from Kameha, he first calls it 'Orb of Rebirth', but then you receive 3 Orbs of Wisdom.
Professor Monster, too, sometimes calls them Orb of Wisdom and Orb of Rebirth. There might be further such instances. Also note that sometimes it's Orb of Wisdom/Rebirth.


Also, on multiple occasions, ellipsises (...) got the spaces wrong.
Sometimes having no space at all (blabla...blabla), sometimes having a space before and after, sometimes having the space before but not after.
It should always be like blabla... bla

**Comment by Clydefrosch22** [comment]
This is a weird one and not really a text issue and likely nothing you can change, but when Murdaw appears in the Arena to offer to breed a monster, he has a Harmour with him, but the monster you actually breed with is Jamirus.

**Comment by Clydefrosch22** [comment]
At the weapon smith, when you get the upgraded selection of possible upgrades, all the skills with + seem to be the inferior one?

ATK Boost+ gives you +1 Attack every 3 levels but
ATK Boost gives you +1 Attack every level up.

Either the effect descriptions are mixed up, or the skill names are.

**Comment by Clydefrosch22** [comment]
![2031-04-12_21-06-22 300_top](https://user-images.githubusercontent.com/21969634/115449562-462b6d80-a21b-11eb-8ae0-e7d8348ad6fd.jpg)
![2031-04-12_21-07-13 794_top](https://user-images.githubusercontent.com/21969634/115449565-475c9a80-a21b-11eb-8d01-fd87960b0d54.jpg)

**Comment by Clydefrosch22** [comment]
Here's a flood of screenshots with notes
https://imgur.com/a/32IcwBs

**Comment by Ich73** [comment]
> there's a lot of messed up line breaks in the wifi lobby, mostly when the 'hometown' is displayed. Various places have linebreaks, I guess everything with more than 6 Letters.

I changed "hometown" to "home". If there is anything more than needs to be changed in the WiFi Lobby, please make some screenshots and create a separate issue for it.

**Comment by Ich73** [comment]
> Every week when the ranked battles are through, the top screen shows a picture saying "Took 1st Place!!" no matter what place you actually ranked as. I suppose that is an image and can't be changed dynamically to display the actual rank, so maybe translate it to something more vague or remove the text at all.

The Japanese image has "上位入賞しました!!" which translates to "I won the top prize!!"
Although it seems out of place, I'll leave it as the original had it.

**Comment by Clydefrosch22** [comment]
> 
> 
> > Every week when the ranked battles are through, the top screen shows a picture saying "Took 1st Place!!" no matter what place you actually ranked as. I suppose that is an image and can't be changed dynamically to display the actual rank, so maybe translate it to something more vague or remove the text at all.
> 
> The Japanese image has "上位入賞しました!!" which translates to "I won the top prize!!"
> Although it seems out of place, I'll leave it as the original had it.

I see. Well, I don't speak the language, but it sounds like it's an expression? You won a top prize/made the rankings, opposed to just a participation badge or something.
It seems you'd have to place like lower than 10000th place to not be ranked. And there's never 10000 people participating from what I can tell xD

**Comment by Clydefrosch22** [comment]
The Red Scrolls description says
Teaches one monster Fighter.
But it actually teaches Fire Fighter.

**Comment by Ich73** [comment]
> i'm actually not sure if what he says here is true? is there a part of rebirth breeding that requires parents to have the same family?

It's incorrect and will be changed with the next commit.
